modules/wkt/docs/api-reference/wkb-writer.md

```diff
@@ -0,0 +1,57 @@
+# WKBLoader
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.2-blue.svg?style=flat-square" alt="From-v3.1" />
+</p>
+
+Writer for the [Well-known binary][wkb] format for representation of geometry.
+
+[wkb]: https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary
+
+| Loader                | Characteristic                                |
+| --------------------- | --------------------------------------------- |
+| File Extension        | `.wkb`,                                       |
+| File Type             | Binary                                        |
+| File Format           | [Well Known Binary][wkb]                      |
+| Data Format           | [Geometry](/docs/specifications/category-gis) |
+| Supported APIs        | `encode`, `encodeSync`                        |
+| Encoder Type          | Synchronous                                   |
+| Worker Thread Support | Yes                                           |
+
+## Usage
+
+```js
+import {WKBWriter} from '@loaders.gl/wkt';
+import {encodeSync} from '@loaders.gl/core';
+
+const geometry = {
+  type: "Polygon",
+  coordinates: [[[1, 2], [3, 4], [5, 6], [1, 2]]]
+}
+const arrayBuffer = encodeSync(geometry, WKBWriter, {wkt: {hasZ: false, hasM: false}})
+```
+
+## Options
+
+- `hasZ`: Should be `true` if the GeoJSON input has Z values. These values are expected to be the third coordinate position.
+- `hasM`: Should be `true` if the GeoJSON input has M values. Thes are expected to be the third coordinate position if Z values do not exist, or fourth if Z values do exist.
+
+## Format Summary
```

Comment:
Nice writeup. Do we duplicate this between the writer and loader pages or consolidate on one of them or the main page for the module?

Reply:
This was copied from the loader page